plugins/commands/reporter/src/main/kotlin/ReporterCommand.kt

```diff
@@ -54,6 +54,7 @@ import org.ossreviewtoolkit.model.readValue
 import org.ossreviewtoolkit.model.readValueOrDefault
 import org.ossreviewtoolkit.model.utils.CompositePackageConfigurationProvider
 import org.ossreviewtoolkit.model.utils.DefaultResolutionProvider
+import org.ossreviewtoolkit.model.utils.setResolutions
```

Comment:
IMO we could merge up to and including this commit, if you want to split the PR to eventually speed to the review process.

Reply:
I've split it out and included one additional commit, as I believe that one to be very much related to this commit: https://github.com/oss-review-toolkit/ort/pull/8048